test/parallel/test-signal-handler.js

```diff
@@ -22,14 +22,14 @@ var i = 0;
 setInterval(function() {
   console.log('running process...' + ++i);
 
-  if (i == 5) {
+  if (i === 5) {
     process.kill(process.pid, 'SIGUSR1');
   }
 }, 1);
 
 // Test on condition where a watcher for SIGNAL
 // has been previously registered, and `process.listeners(SIGNAL).length === 1`
-process.on('SIGHUP', function() {});
+process.on('SIGHUP', function() { common.fail('should not run'); });
```

Comment:
If the function is called, the test has failed. It would be wrapped in a 'common.mustNotCall()' if we had one.